keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -213,177 +213,190 @@ spec:
 ```
-1. `principal`: username/access-key for the object storage provider to uniquely identify the user who has access to this bucket. This value is minted by the provisioner (and set in the BA) when the `BucketAccessRequest` omits a ServiceAccount. There are several use cases involving `principal`: 
-   - Not relevant if serviceAccount is supplied.
-   - For brownfield cases when there is already a storage-side principal that you want to use for access, the admin can supply this, the driver will (possibly) enable access for that principal to the bucket in question and will return credentials.
-   - For greenfield cases, this could be populated by the driver. Maybe so that the driver can request deletion of this principal during deprovisioning
-1. `parameters`:  A map of string:string key values.  Allows admins to control user and access provisioning by setting provisioner key-values. Copied from `BucketAccessClass`. 
-1. `accessGranted`: if true access has been granted to the bucket. If false then access to the bucket has not been granted.
+10. `parameters`:  A map of string:string key values.  Allows admins to control user and access provisioning by setting provisioner key-values. Copied from `BucketAccessClass`. 
+11. `accessGranted`: if true access has been granted to the bucket. If false then access to the bucket has not been granted.
+12. `credentials`: a `SecretReference` to the sidecar-generated Secret containing access credentials. This secret exists in the provisioner’s namespace, is read by the cosi-node-adapter, and written to the secret mount defined in the app pod's `csi-driver spec`.
+13. `accountID`: username/access-key for the object storage provider that identifies the user with access to this bucket. This value is minted by the driver when the `BucketAccessRequest` omits a ServiceAccount.
 
 #### BucketAccessClass
 
-An immutable, cluster-scoped, custom resource providing a way for admins to specify policies that may be used to access buckets. A `BucketAccessClass` (BAC) is required for both greenfield and brownfield use cases. Besides naming the provisioner, a BAC references a config map which is expected to define the access policy for a given provider. It is typical for these config maps to reside in each provisioner's namespace, though this is not required.  Unlike BucketClasses, there is no default BAC.
+An immutable, cluster-scoped, brownfield-only, custom resource providing a way for admins to specify policies used to access buckets. A `BucketAccessClass` (BAC) is referenced by a user [BucketAccessRequest](#bucketaccessrequest) and is used to populate the [BucketAccess](#bucketaccess) instance. A BAC references a config map which is expected to define the access policy for a given provider. It is typical for these config maps to reside in each provisioner's namespace, though this is not required. Unlike a [BucketClass](#bucketclass), there is no default BAC.
```

Comment:
I'm likely missing something obvious, but I was surprised to read a BAC is brownfield only? Could this be explained in the doc?

Reply:
I also just learned that BAC is brownfield only.  Please explain why.

If this is really brownfield only, then I suggest we use a different name, not "Class".  "Class" such as StorageClass and VolumeSnapshotClass are associated with dynamically provisioned resources (greenfield).

Reply:
> I'm likely missing something obvious, but I was surprised to read a BAC is brownfield only? Could this be explained in the doc?

BAC is definitely not Brownfield only. Might be a mistake carried over from an older design. @jeffvance --^